docs/content/integrations/looker.mdx

```diff
@@ -57,6 +57,36 @@ looker_specs = load_looker_asset_specs(looker_resource=looker_resource)
 defs = dg.Definitions(assets=[*looker_specs], resources={"looker": looker_resource})
 ```
 
+## Load Looker assets from filtered dashboards and explores
+
+It is possible to load a subset of your Looker assets by providing a <PyObject module="dagster_looker" method="LookerFilter" /> to the <PyObject module="dagster_looker" method="load_looker_asset_specs" /> function. All dashboards contained in the folders provided to your <PyObject module="dagster_looker" method="LookerFilter" /> will be fetched. Additionally, only the explores used in these dashboards will be fetched by passing `only_fetch_explores_used_in_dashboards=True` to you to your <PyObject module="dagster_looker" method="LookerFilter" />.
```

Comment:
@maximearmstrong one minor nit, looks like you accidentally duplicated the "to you" in:
> ... will be fetched by passing `only_fetch_explores_used_in_dashboards=True` **to you to your** LookerFilter.